sales_team_multicompany/security/ir.rule.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,groups:id,perm_read,perm_write,perm_create,perm_unlink,domain_force
+sales_team.crm_team_rule,crm.team multi-company,sales_team.model_crm_team,,True,False,False,False,"['|', ('company_ids', '=', 'user.company_id.id'), ('company_ids','child_of',[user.company_id.id])]"
```

Comment:
The rules are usually put in XML and with a noupdate="1" attribute for allowing to change the rule manually.